trillian/ctfe/instance.go

```diff
@@ -75,17 +76,13 @@ type Instance struct {
 // up-to-date with any tree head changes that are not triggered by us.
 func (i *Instance) RunUpdateSTH(ctx context.Context, period time.Duration) {
 	c := i.li.instanceOpts.Validated.Config
-	ticker := time.NewTicker(period)
 	glog.Infof("Start internal get-sth operations on %v (%d)", c.Prefix, c.LogId)
-	for t := range ticker.C {
-		glog.V(1).Infof("Tick at %v: force internal get-sth for %v (%d)", t, c.Prefix, c.LogId)
+	schedule.Every(ctx, period, func(ctx context.Context) {
```

Comment:
Similarly, behaviour slightly changes here in that a get-sth call will be made immediately, rather than waiting `period` before making the first call. This seems fine though.